PowerShellTools.HostService/ServiceManagement/Debugging/PowerShellDebuggingService.cs

```diff
@@ -132,6 +144,124 @@ public void SetRunspace(bool overrideExecutionPolicy)
         }
 
         /// <summary>
+        /// Attaches the HostService to a local runspace already in execution
+        /// </summary>
+        public void AttachToRunspace(uint pid)
+        {
+            if (_callback == null)
+            {
+                _callback = OperationContext.Current.GetCallbackChannel<IDebugEngineCallback>();
+            }
+
+            // Attaching leverages cmdlets introduced in PSv5
+            if (_installedPowerShellVersion < RequiredPowerShellVersionForProcessAttach)
+            {
+                MessageBox.Show(Constants.ProcessAttachVersionError, "Invalid Version", MessageBoxButton.OK, MessageBoxImage.Warning);

```

Comment:
Put raw string into constant, and we might want to specifically demonstrate that it is just the poshtools debugger is not able to attached to process, coz I think there is nothing prevent user attach other debugger onto the process. so this message needs to be more specific about powershell.